ACS Calling Wpf App Sample

chriswhilar opened the pull request:
## Purpose
<!-- Describe the intention of the changes being proposed. What problem does it solve or functionality does it add? -->
ACS Calling Sample for WPF

## Does this introduce a breaking change?
<!-- Mark one with an "x". -->
```
[ ] Yes
[ X ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

<!-- Please check the one that applies to this PR using "x". -->
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ X ] Other... Please describe:
New Code
```

## How to Test
```
Follow the README
```

chriswhilar committed:
ACS Calling Wpf App Sample

LoadLibrary commented on README.md:
```diff
@@ -1,53 +1,41 @@
 # Project Name
 
-(short, 1-3 sentenced, description of the project)
+ACS Calling Sample for WPF.
```
It would be better informing in the description that this sample app uses WebView2 and the Web SKU of the Calling SDK.

LoadLibrary commented on README.md:
```diff
@@ -1,53 +1,41 @@
+- [Node.js](https://nodejs.org/en/download/)
+- [Microsoft Edge (Chromium) Canary Chanel](https://www.microsoftedgeinsider.com/en-us/download)
+- [Visual Studio 2017 or later](https://visualstudio.microsoft.com/)
+- .NET Framework. It can be installed from Visual Studio Installer.
```
It would be better inform that the VS workload required: Microsoft.VisualStudio.Workload.ManagedDesktop

https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-enterprise?view=vs-2019&preserve-view=true#net-desktop-development

LoadLibrary commented on README.md:
```diff
@@ -1,53 +1,41 @@
+- [npm](https://www.npmjs.com/get-npm)
+- [Node.js](https://nodejs.org/en/download/)
+- [Microsoft Edge (Chromium) Canary Chanel](https://www.microsoftedgeinsider.com/en-us/download)
+- [Visual Studio 2017 or later](https://visualstudio.microsoft.com/)
```
Have you test this project on VS 2017?

chriswhilar replied:
Yes it does, I just tested it.

LoadLibrary reviewed (changes requested):
README.md requires some changes to help devs to understand what the project does and how to have better chances to have it running in their machines.

akania commented on README.md:
```diff
@@ -1,53 +1,41 @@
-2.
-3.
-
+1. You will first need to set up and run the ACS Calling Tutorial project on localhost:5000. To do this, please follow the README file on the repo https://github.com/Azure-Samples/communication-services-web-calling-tutorial 
```
nit, remote "You will first need to", start form "Setup and run... "

akania commented on README.md:
```diff
@@ -1,53 +1,41 @@
+1. You will first need to set up and run the ACS Calling Tutorial project on localhost:5000. To do this, please follow the README file on the repo https://github.com/Azure-Samples/communication-services-web-calling-tutorial 
+2. git clone https://github.com/Azure-Samples/communication-services-web-calling-wpf-sample
+3. cd project
+4. Open WpfApp.sln in Visual Studio
```
nit, I assume one has to "Open WpfApp.sln from `project` directory of a cloned repository"

akania commented on README.md:
```diff
@@ -1,53 +1,41 @@
+![Manage Nuget packages](./images/ManageNuget.png)
+6. From NuGet dialog, browse for Microsoft.Web.WebView2 (Make sure to check the "Include prerelease" box) and install version 0.9.579-prerelease
+![Nuget Dialog](./images/NugetDialog.png)
+7. Start the WPF app from Visual Studio and the WPF app should come up. This WPF app has two web views. In each web view you should see the ACS Calling Tutorial application
```
nit, Start -> Build and Run..

akania commented on README.md:
```diff
@@ -1,53 +1,41 @@
+![Nuget Dialog](./images/NugetDialog.png)
+7. Start the WPF app from Visual Studio and the WPF app should come up. This WPF app has two web views. In each web view you should see the ACS Calling Tutorial application
+![Wpf App](./images/WpfApp.png)
+8. From the ACS Calling Tutorial applications, click on the orange "Provision user and initialize SDK" buttons. You will now be able to do calls. Click on the orange "Join group" buttons for both ACS Calling applications to joing a group call:
```
nit, detail but what if we change color schema and there won't be 'orange' button :)

akania replied:
buttons -> button ?

akania commented on README.md:
```diff
@@ -1,53 +1,41 @@
+
+## Troubleshooting
+1. If the WpfApp shows up blank, reinstall the Microsoft.Web.WebView2 SDK version 0.9.579-prerelease from the Nuget dialogue.
+2. Please post any issues, and the ACS Calling team will reply and look into any issues as soon as possible
```
nit, provide link?

chriswhilar committed:
Address Augusto's and Artur's review comments

chriswhilar commented:
Iv' addressed all comments from Artur and Augusto

chriswhilar committed:
Add simple calling web app and update readme

chriswhilar committed:
remove call end reason

chriswhilar committed:
commit images

chriswhilar commented:
@mariusu-msft @akania @LoadLibrary Hey folks, i have added a super simple calling web app so now this sample is stand alone and I have updated the read me. Please pull my branch, try it out, and review.
